### PR TITLE
DCD-1158: add snyk into action

### PR DESCRIPTION
**Pull request by XutingQ**

### CODE REVIEW

**Comment by nanux** [comment]
Seems there is an issue with authentication
```
 `snyk` requires an authenticated account. Please run `snyk auth` and try again.
```

**Review by nanux — COMMENTED** [review]
Ca we also add some basic documentation into the `README.md` file?

File: .github/workflows/snyk_security.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Security
```
[the remark this answers is not in the document]
Comment (XutingQ) [reply .github/workflows/snyk_security.yml:1]: ok

**Comment by XutingQ** [comment]
> Ca we also add some basic documentation into the `README.md` file?

What kind doc do you want to add into `README.md`?

**Comment by nanux** [comment]
> > Ca we also add some basic documentation into the `README.md` file?
> 
> What kind doc do you want to add into `README.md`?

Something along the lines:

```
### Security scan

The project is regularly scanned for security vulnerabilities via Snyk integration.
```

**Review by nanux — APPROVED** [review]

**Pull request merged**